### ISSUE TITLE
Consistent button SVG fill color and background

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/sul-embed#1874]
The top-level navigation buttons should have the same SVG fill color (and hover background color) as the Sidebar tab buttons.
<img width="400" alt="svg-fill-color" src="https://github.com/sul-dlss/sul-embed/assets/2343316/145a0a8c-80a1-416c-9102-d9825604580c" />
Update the icon fill and hover background colors to meet WCAG AA color contrast requirements, as follows.
<img width="400" alt="wcag-button-color" src="https://github.com/sul-dlss/sul-embed/assets/2343316/b48839ca-61f7-48b6-ad93-ab7f62d03fc9" />
NB: Keep the current cardinal red fill color (`rgb(140, 21, 21)`) for buttons in an active state.
